src/frontend/js/modules/session.js

```diff
@@ -121,6 +121,10 @@
             });
         })
         .catch(error => {
+          if (error.code === 401) {
```

Comment:
Shouldnt we check if error.code is defined?

Reply:
If not best practice is Always to test static value against variable. Ex if (401 === error.code)